google/api_core/grpc_helpers_async.py

```diff
@@ -207,10 +196,17 @@ def wrap_errors(callable_):
 
     Returns: Callable: The wrapped gRPC callable.
     """
+    grpc_helpers._patch_callable_name(callable_)
     if isinstance(callable_, aio.UnaryUnaryMultiCallable):
         return _wrap_unary_errors(callable_)
+    elif isinstance(callable_, aio.UnaryStreamMultiCallable):
+        return _wrap_stream_errors(callable_, _WrappedUnaryStreamCall)
+    elif isinstance(callable_, aio.StreamUnaryMultiCallable):
+        return _wrap_stream_errors(callable_, _WrappedStreamUnaryCall)
+    elif isinstance(callable_, aio.StreamStreamMultiCallable):
+        return _wrap_stream_errors(callable_, _WrappedStreamStreamCall)
     else:
-        return _wrap_stream_errors(callable_)
+        raise TypeError("Unexpected type of callable: {}".format(type(callable_)))
```

Comment:
nit (not a blocker, consider for the future): consider in cases like this raising a custom subclass of `TypeError`, so that we can check the exception type in tests